picard/ui/metadatabox/__init__.py

```diff
@@ -282,19 +285,61 @@ def _copy_value(self):
                 except (TypeError, ValueError) as why:
                     log.warning(why)
                     value = ['']
-
-            if value is not None:
-                self.tagger.clipboard().setText(MULTI_VALUED_JOINER.join(value))
-                self.clipboard = value
+            return value
+
+        #TODO: Support  copying data from multiple files
```

Comment:
I'm not even sure this should be implemented. What would be the semantics when pasting the data? Right now it would skip values that are different, and I think it is fine. The only other way would be to combine all values into a list. But not sure if this is even a usecase. If, then maybe as an explicit action on those tags to copy all values from all files.

Reply:
Agreed